products/opensuse/main.pm

```diff
@@ -348,10 +348,22 @@ else {
         load_reboot_tests();
     }
     elsif (installzdupstep_is_applicable()) {
+        if (is_upgrade && get_var('PATCH_BEFORE_MIGRATION')) {
```

Comment:
I don't see this code path in any VR

Reply:
https://openqa.opensuse.org/tests/overview?distri=opensuse&version=Tumbleweed&build=20240521&groupid=119

Reply:
for zdup please check https://openqa.opensuse.org/tests/4214694